main.py

```diff
@@ -1,12 +1,14 @@
 """ToDo."""
-from tasks.kyu8 import example
+from tasks.kyu6 import approximation
 
 
-def run_example():
-    """ToDo"""
-    first = input("a: ")
-    second = input("b: ")
-    print(example(first, second))
+def run_approximation():
+    """ run approximation """
+    try:
+        num = int(input("number: "))
+    except ValueError:
+        print('Wrong input')
+    print(approximation(num))
```

Comment:
This should be in try-block because there is a chance that you will call function passing wrong input parameter.